code/game/objects/items/toys.dm

```diff
@@ -870,58 +870,45 @@
 	w_class = WEIGHT_CLASS_TINY
 	var/list/currenthand = list()
 	var/choice = null
-
+	var/list/handradial = list()
```

Comment:
Can you do a lazy list and document it?

Reply:
As we discussed on the discord, I moved the var into the proc it's used for rather than making a lazy list for this situation.